include/openssl/rand_drbg.h

```diff
@@ -22,7 +22,18 @@
 
 /* In CTR mode, disable derivation function ctr_df */
 # define RAND_DRBG_FLAG_CTR_NO_DF            0x1
+/*
+ * This flag is only used when a digest NID is specified (i.e: not a CTR cipher)
+ * Selects DRBG_HMAC if this is set otherwise use DRBG_HASH.
+ */
+# define RAND_DRBG_FLAG_HMAC                 0x2
```

Comment:
Why no `0x04`or `0x08` flag?

Reply:
I have moved them down - I wanted to logically seperate the last 3 flags from the other flags - but it doesnt really matter.

Reply:
I think I will leave the RANDerr call alone for now.. I think this is probably an issue across the entire code base that needs to be addressed.